### DISCUSSION TITLE
Gopro 4 look

### DISCUSSION THREAD

**Discussion by lyricsdude** [gopro/labs#970] [General]
I like the look of the Gopro 4. The shadows are not so dark. How can I achieve this look?

**Comment by lyricsdude** [comment]
https://www.youtube.com/watch?v=4hhKamFXHXo

For example this video that I just saw.

It looks so much more realistic.

**Comment by dnewman-gpsw** [comment]
Just use the Flat profile, as you can get that look more easily from there.
